### PR TITLE
add i18n-keys to project for testing i18n key availability and use

### PR DESCRIPTION
**Pull request by mejackreed**
Adds to the CI build automated checking of i18n key comparisons.

### CODE REVIEW

File: config/i18n-tasks.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# i18n-tasks finds and manages missing and unused translations: https://github.com/glebm/i18n-tasks
+
+# The "main" locale.
+base_locale: en
+## All available locales are inferred from the data by default. Alternatively, specify them explicitly:
+locales: [es, fr, de, it, pt-BR]
```
Comment (cbeer) [inline config/i18n-tasks.yml]: Inferred locales don't work because we're in an engine?

File: blacklight.gemspec [hunk 2]
```diff
@@ -46,4 +46,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency "simplecov"
   s.add_development_dependency "rubocop"
   s.add_development_dependency "rubocop-rspec"
+  s.add_development_dependency "i18n-tasks", "~> 0.9.5"
```
Comment (cbeer) [inline blacklight.gemspec]: Does it need to be this pessimistic?

File: config/i18n-tasks.yml (hunk 1 above)
Comment (mejackreed) [inline config/i18n-tasks.yml]: I think so? Tried them but it didn't pick them up.

File: blacklight.gemspec (hunk 2 above)
Comment (mejackreed) [inline blacklight.gemspec]: Probably not. Push in a separate commit?

File: config/i18n-tasks.yml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  # Locale files or `File.find` patterns where translations are read from:
+  read:
+    ## Default:
+    - config/locales/%{locale}.yml
+    ## More files:
+    - config/locales/**/*.%{locale}.yml
+    ## Another gem (replace %#= with %=):
+    # - "<%#= %x[bundle show vagrant].chomp %>/templates/locales/%{locale}.yml"
+
+  # Locale files to write new keys to, based on a list of key pattern => file rules. Matched from top to bottom:
```
Comment (cbeer) [inline config/i18n-tasks.yml]: Can we clean up some of these default comments? I think it'd be easier to see our specific guidelines without these.

Comment (cbeer) [inline config/i18n-tasks.yml]: Ok, sounds good.

**Pull request merged**